addons/loyalty/models/loyalty_reward.py

```diff
@@ -105,6 +106,10 @@ def _compute_reward_product_uom_id(self):
         for reward in self:
             reward.reward_product_uom_id = reward.reward_product_ids.product_tmpl_id.uom_id[:1]
 
+    def _search_reward_product_ids(self, operator, value):
+        assert operator in ('=', '!=', 'in')
```

Comment:
it's a detail, but I think in similar use case we usually do `if  ...: raise NotImplementedError()` instead of an AssertionError.